libs/sdk-ui-dashboard/src/model/store/user/userSelectors.ts

```diff
@@ -19,3 +19,13 @@ export const selectUser = createSelector(selectSelf, (state) => {
 
     return state.user!;
 });
+
+/**
+ * This selector returns current logged in user ref. It is expected that the
+ * selector is called only after the permission state is correctly initialized. Invocations before initialization lead to invariant errors.
+ *
+ * @alpha
+ */
+export const selectUserRef = createSelector(selectUser, (user) => {
```

Comment:
i suggest rename to selectCurrentUserRef.. be somewhat more concrete

Reply:
Fixed renamed to selectCurrentUserRef